src/lms-ds-loader/edfi_lms_ds_loader/mssql_lms_operations.py

```diff
@@ -136,21 +138,91 @@ def insert_new_records_to_production(self, table: str, columns: List[str]) -> No
         assert table.strip() != "", "Argument `table` cannot be whitespace"
         assert len(columns) > 0, "Argument `columns` cannot be empty"
 
-        column_string = ", ".join([f"[{c}]" for c in columns])
+        column_string = ",".join([f"\n    {c}" for c in columns])
 
         statement = f"""
-insert into lms.[{table}] ( {column_string} )
-select {column_string}
-from lms.stg_{table} as stg
-where not exists (
-  select 1 from lms.[{table}]
-  where sourcesystemidentifier = stg.sourcesystemidentifier
-  and sourcesystem = stg.sourcesystem
+INSERT INTO
```

Comment:
I reformatted these statements into a form I have been using with Analytics Middle Tier, which I think it generally very easy to read. The above was previously easy to read, but I think this expanded format is better - especially when you after the `{column_string}` substitution, and you look at the actual SQL statement that is sent to the server.